clash-lib/tools/static-files.hs

```diff
@@ -0,0 +1,69 @@
+{-|
+Copyright   :  (C) 2022     , Google Inc.
+License     :  BSD2 (see the file LICENSE)
+Maintainer  :  QBayLogic B.V. <devops@qbaylogic.com>
+
+Produce static files that are useful when working with Clash designs.
+-}
+
+{-# LANGUAGE QuasiQuotes #-}
+
+module Main where
+
+import Control.Monad (when)
+import Control.Monad.Extra (whenM, unlessM)
+import Prelude
+import System.Console.Docopt
+  (Docopt, docopt, isPresent, getArg, longOption, parseArgsOrExit)
+import System.Directory (copyFile, doesDirectoryExist, doesFileExist)
+import System.Environment (getArgs)
+import System.Exit (die)
+import System.FilePath ((</>), takeDirectory)
+
+import Paths_clash_lib (getDataFileName)
+
+patterns :: Docopt
+patterns = [docopt|
+Obtain static files useful when working with Clash designs.
+
+Currently, only the Tcl connector is available.
+
+Usage:
+  static-files [--force] --tcl-connector=<outfile>
+
+Options:
+  -f, --force                  Overwrite existing files
+  --tcl-connector=<outfile>    Write a Tcl script to a file that can parse Clash
```

Comment:
Given that it's for Vivado specifically, I'd mention it in the name.

Reply:
I was planning to extend it to other tools some day, and it can already be used on Intel when you just do `clash::readMetadata` and then parse the `metadata` dict yourself. But if you still think it should mention Vivado specifically, I'll add it.